src/pages/image/_id.vue

```diff
@@ -97,6 +97,7 @@ import VImageDetails from '~/components/VImageDetails/VImageDetails.vue'
 import VMediaReuse from '~/components/VMediaInfo/VMediaReuse.vue'
 import VRelatedImages from '~/components/VImageDetails/VRelatedImages.vue'
 import SketchFabViewer from '~/components/SketchFabViewer.vue'
+import { IMAGE } from '@/constants/media'
```

Comment:
I'm going to go ahead and add the import eslint rules today to prevent these.

```suggestion
import { IMAGE } from '~/constants/media'
```